bin/reth/src/prometheus_exporter.rs

```diff
@@ -74,54 +74,18 @@ async fn start_endpoint<F: Hook + 'static>(
 }
 
 /// Serves Prometheus metrics over HTTP with database and process metrics.
-pub(crate) async fn serve(
+pub(crate) async fn serve<DB: DatabaseMetrics + 'static>(
```

Comment:
I'd prefer a where close here instead
```suggestion
pub(crate) async fn serve<Metrics>(
```

Reply:
using a where clause now